helm/nostr/strfry-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: strfry
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: strfry
+  template:
+    metadata:
+      labels:
+        app: strfry
+    spec:
+      containers:
+        - name: strfry
+          image: dockurr/strfry
+          ports:
+            - containerPort: 7777
+          volumeMounts:
+            - name: strfry-db
+              mountPath: /app/strfry-db
+            - name: strfry-conf
+              mountPath: /etc/store.conf
+      volumes:
+        - name: strfry-db
+          persistentVolumeClaim:
+            claimName: strfry-chart-db-pvc
+        - name: strfry-conf
+          configMap:
+            name: strfry-chart-configmap
```

Comment:
let's use the same definition used in configmap. i.e `{{ include "strfry-chart.fullname" . }}-configmap`

Right now, since this is hardcoded, this would deploy if someone tried to provide an alternative name